zephyr/module.yml

```diff
@@ -17,3 +17,10 @@ blobs:
     license-path: zephyr/blobs/stm32wba/lib/license.md
     url: https://github.com/STMicroelectronics/STM32CubeWBA/raw/v1.3.1/Middlewares/ST/STM32_WPAN/ble/stack/lib/stm32wba_ble_stack_llo.a
     description: "Binary Stack library for the STM32WBA Bluetooth subsystem"
+  - path: ble-fw/steval-mkboxpro/dtm.bin
```

Comment:
Yes, it could :)
It seems to imply that there might be other per-board per-protocol blob images. Which it might be the case indeed sometimes in the future.

Reply:
Actually I have a doubt here. Where should I put the license in this case? According to your suggestion it seems that every board will have it's own license file, I cannot keep same license under board:

- zephyr/blobs/board/steval-mkboxpro/ble-fw/license.md
- zephyr/blobs/board/steval-mksbox1v1/ble-fw/license.md

It's not something that I really like,,,

Reply:
@erwango I left this part as it was. Please help me to clarify my doubt (i.e. using 'boards' and then 'ble-fw' suggests to the user that there will be other per-board per-protocol blob images, which is not the intention of this PR)

Reply:
My issue with current proposal is that it doesn't fit with the existing tree which stores what can be called ble-fw under `blob/wba/lib`.
So I do think that we should keep `boards` but ok to remove `ble-fw`.
Then maybe we'll have to move `wba` under a new `soc` root.

Reply:
@erwango Hey Erwan, just restarted to work on this since very long time. OK, I think that we can proceed as you suggested. Under ```blobs``` I'll create ```boards``` and maybe you will add ```soc``` as for your convenience.